src/AppBundle/Form/CommentType.php

```diff
@@ -54,9 +47,6 @@ public function configureOptions(OptionsResolver $resolver)
         ));
     }
 
-    /**
-     * @return string
-     */
     public function getName()
```

Comment:
This PR is about removing docblocks, not adding `{@inheritdoc}`. I just command about changed made here.